modules/core-updates/9.0.md

```diff
@@ -85,6 +85,40 @@ Some dependencies are still present but were upgraded which comes with their own
 - [Services translator, security.csrf.token_manager, session, session.flash_bag no longer public](https://github.com/PrestaShop/PrestaShop/pull/35069)
 - [Replace AdminSecurity annotation with attributes](https://github.com/PrestaShop/PrestaShop/pull/35212)
 
+### BO login and authorization migrated to Symfony
+
+The back office login page has been migrated to Symfony. Along with this change, the authorization system in the back office is now also based on Symfony, which implies several things:
+- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side, and very few data are kept on the browser side, for retro-compatibility we still populate the legacy cookie so that you can **read** it Although you can modify it for your custom data but modifying the data previously used for PrestaShop authorization will no longer have effect (or unstable ones) so it's not recommended
```

Comment:
```suggestion
- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side, and very few data are kept on the browser side. For retro-compatibility, we still populate the legacy cookie so that you can **read** it. While you can work with `Context::$cookie` and your custom data, making changes to the data used for PrestaShop authorization may no longer have the desired effect or may result in instability, so it's not recommended.
```

A little rephrase.